.gitignore

```diff
@@ -55,3 +55,5 @@ nbactions.xml
 .idea/
 *.iml
 *.iws
+gateway.properties
+Product/Production/Common/Properties/localhost.cer
```

Comment:
Is it only this particular localhost.cer that should be ignored?  Otherwise, the path before it can be removed.